inst/ms/aggreCAT.qmd

```diff
@@ -1244,8 +1331,10 @@ replicate, forecasts were inaccurate, with `IndIntWAgg`, `IntWAgg` and
 #| fig-align: center
 #| message: false
 #| warning: false
-#| fig-cap: "Blocked heatmap visualisation of confidence scores is useful for visually comparing aggregation methods and evaluating them against a set of known outcomes. In this example, Confidence Scores generated by 6 aggregation methods for the repliCATS pilot study are visualised for 25 claims. Claims where known outcomes succesfully replicated (outcome == `TRUE`) are presented in heatmap blocks on the left, and claims that failed to replicate are presented in heatmap blocks on the right. Confidence Scores generated by different aggregation methods are positioned along the y-axis, with vertical groupings according to the methods' mathematical properties. Colour and intensity of cells indicates the direction and degree of deviation respectively of the Confidence Scores from the known outcomes."
 #| eval: true
+#| fig-cap: "Blocked heatmap visualisation of confidence scores is useful for visually comparing aggregation methods and evaluating them against a set of known outcomes. In this example, Confidence Scores generated by 6 aggregation methods for the repliCATS pilot study are visualised for 25 claims. Claims where known outcomes succesfully replicated `outcome == TRUE` are presented in heatmap blocks on the left, and claims that failed to replicate are presented in heatmap blocks on the right. Confidence Scores generated by different aggregation methods are positioned along the y-axis, with vertical groupings according to the methods' mathematical properties. Colour and intensity of cells indicates the direction and degree of deviation respectively of the Confidence Scores from the known outcomes."
```

Comment:
This figure is _waaaaayyy_ better than the previous one. But I have a few suggestions still:
- I don't think the supp data methods should be their own facet. They're different from a code implementation perspective, but not statistically, and when you're visualising the results you're not considering code implementation anymore.
- As per previous plot, go back to base `theme_bw()` and keep facet label boxes, also drop italics inf facet labels. And is the legend title in a different font?

Reply:
Agree with both points. New issue created #34

Reply:
@psychtek was working on this, is there PR or is it on aggrecat-dev?